internal/cmd/base/servers.go

```diff
@@ -84,6 +84,7 @@ type Server struct {
 
```

Comment:
In `internal/cmd/base/servers.go` can you add the new kms to `SetupKMSes`
And also add to the test `TestServer_SetupKMSes` in  `internal/cmd/base/server_test.go`